examples/smbexec.py

```diff
@@ -56,7 +56,7 @@
 from impacket.dcerpc.v5 import transport, scmr
 from impacket.krb5.keytab import Keytab
 
-OUTPUT_FILENAME = '__output'
+OUTPUT_FILENAME = '__output'.join(random.sample(['z','y','x','w','v','u','t','s','r','q','p','o','n','m','l','k','j','i','h','g','f','e','d','c','b','a'], 2))
```

Comment:
It would be better to `import string` and then use `string.ascii_lowercase` as follows
```suggestion
OUTPUT_FILENAME = '__output'.join(random.sample(string.ascii_lowercase, 2))
```